src/containers/home/index.js

```diff
@@ -4,10 +4,8 @@ import { push } from 'react-router-redux';
 import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import styled, { css } from 'styled-components';
-
 import * as actions from './actions';
```

Comment:
missing space above, you should group your imports. 

imports from node_modules
*SPACE*
import your own stuff